scripts/ASTImportTest.sh

```diff
@@ -40,11 +40,11 @@ function testImportExportEquivalence {
         $SOLC --import-ast --combined-json ast,compact-format --pretty-json expected.json > obtained.json 2> /dev/null
         if [ $? -ne 0 ]
         then
-               # For investigating, use exit 1 here so the script stops at the
-               # first failing test
-               # exit 1
-               FAILED=$((FAILED + 1))
-               return 1
+           # For investigating, use exit 1 here so the script stops at the
```

Comment:
These seem to be 3 spaces in instead of 4 ?

Reply:
Right. Sorry for that. I quickly reviewed the diff again and I don't see any other mistakes like this so it should be fine now.